src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -76,6 +76,12 @@
     text: az appconfig kv export -n MyAppConfiguration -d appservice  --appservice-account MyAppService
   - name: Export all keys with label test excluding feature flags to a json file.
     text: az appconfig kv export -n MyAppConfiguration --label test -d file --path D:/abc.json --format json --skip-features
+  - name: Export all keys and feature flags with all labels to another App Configuration.
+    text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --preserve-labels
+  - name: Export all keys and feature flags with all labels to another App Configuration and overwrite destination labels.
+    text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --dest-label ExportedKeys
+  - name: Export all keys and feature flags with all labels to another App Configuration and assign null label in destination AppConfig.
+    text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label *
```

Comment:
nit: Can we just leave ```az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --preserve-labels``` . For the last two new added examples, we can modify the existing one ```az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration``` 
Just don't want to make the help messages too long. Same for import examples.